src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (num) {
+      return this.result + num;
+    },
+    subtract: function (num) {
+      return this.result - num;
+    },
+    multiply: function (num) {
+      return this.result * num;
+    },
+    divide: function (num) {
+      return this.result / num;
+    },
+    reset: function (num) {
+      this.result = 0;
+
+      return this;
+    },
```

Comment:
The `reset` method should not accept any parameters since it's only purpose is to reset the `result` to 0. Remove the unused parameter `num` from the `reset` method signature.